.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
```

Comment:
Fantastic – I believe we essentially use this on Iodide, so this is welcome (for me)